lensterxyz/lenster
# fix: remove paragraphs from editor

## What does this PR do?

Fixes new problem as described in comment https://github.com/lensterxyz/lenster/issues/1193#issuecomment-1420805409

This works by modifying the editor's content tree, replacing all paragraphs with newlines, before the content is exported to markdown

See also https://lexical.dev/docs/concepts/editor-state#updating-state

## Type of change


## How should this be tested?

- Open publication editor, **paste** the content below and publish the post

```
aaaaaa
bbbb

foo
bar
```

- Verify that the published post looks the same

- Repeat the above steps, but typing the content by hand, instead of pasting it
